USACO dec15-feast-gold.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: usaco-574

```

Comment:
yeah, this file isn't in the solutions folder :eyes:

Reply:
> Delete this file? It's duplicated with solutions/(file with same name)

What do you mean? I added the C++ implementation for this problem because there isn't one (the official analysis uses Java).  👀

Reply:
Wait, I create duplicate versions?
Oh sorry, it's my mistake, I'm going to delete this now